expression/builtin.go

```diff
@@ -276,8 +281,16 @@ func newBaseBuiltinCastFunc(builtinFunc baseBuiltinFunc, inUnion bool) baseBuilt
 	}
 }
 
+// vectorizedBuiltinFunc contains all vectorized methods for a builtin function.
+type vectorizedBuiltinFunc interface {
```

Comment:
how about s/vectorizedBuiltinFunc/vecBuiltinFunc/? it's much shorter.

Reply:
fixed.